modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/report/NumericDDMFormFieldTypeReportProcessorTest.java

```diff
@@ -251,6 +251,64 @@ public void testProcessDDMFormInstanceReportWithExistingData()
 		Assert.assertEquals("4", summaryJSONObject.getString("sum"));
 	}
 
+	@Test
+	public void testProcessDDMFormInstanceReportWithLongFieldValue()
```

Comment:
Great job, @marcosapmf! In a very near future, I'll propose to you guys always add a test for every fixed bug.